Makefile

```diff
@@ -53,14 +53,8 @@ examples:
 examples-clean: $(INSTALLDIR)/include/n64.mk
 	$(MAKE) -C examples clean
 
-doxygen: doxygen.conf
-	mkdir -p doxygen/
-	doxygen doxygen.conf
 doxygen-api: doxygen-public.conf
-	mkdir -p doxygen/
 	doxygen doxygen-public.conf
-doxygen-clean:
-	rm -rf $(CURDIR)/doxygen
```

Comment:
Any reason why we should remove doxygen-clean? It doesn't seem wrong to keep it and also have `clobber` depend on it. We use `clobber` as a `clean all` target.

Reply:
the doxygen is generated to website/, afaik the "doxygen" folder is unused besides in this makefile
I could update doxygen-clean to clean "website" though.

Additionally note build.sh runs `make clobber` at the end, but deleting doxygen docs at that point sounds undesirable

Reply:
Ok let's keep it like this